src/encoded/schemas/quality_metric.json

```diff
@@ -55,7 +56,8 @@
                     "title": "MIME type",
                     "type": "string",
                     "enum": [
-                        "text/plain"
+                        "text/plain",
+                        "image/png"
```

Comment:
Mixins are just a convenience for constructing the final schemas. If there are only a few quality metrics that support image, we should leave the default be text and override the attachment property only on those. If it's common then making it generic like this is fine. Being overly restrictive about the attachment type probably doesn't buy us much.